src/python/bot/tasks/corpus_pruning_task.py

```diff
@@ -826,7 +826,7 @@ def execute_task(full_fuzzer_name, job_type):
     _save_coverage_information(context, result)
     _process_corpus_crashes(context, result)
   except CorpusPruningException as e:
-    logs.log_error('Corpus pruning failed: %s.' % str(e))
```

Comment:
This shouldn't be necessary since we'll log the exception info regardless. The issue in both cases is that the exception information isn't necessarily valid unicode, so when we try to convert it to a newstr it can lead to an exception. This will probably need more thought as we try to get everything running properly in Python 3, but for now this seems like a quick exception fix.